### PR TITLE
I add tcpdump monitor python code

### PR DESCRIPTION
**Pull request by cklonger**
python of source need to review

### CODE REVIEW

File: honeypod/TcpdumpMonitor.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import subprocess
+import os
+
```
Comment (storkinsj) [inline honeypod/TcpdumpMonitor.py:6]: Please pull required changes from p0f into this module:
import sys
import psutil
from socket import AF_INET

File: honeypod/TcpdumpMonitor.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+class TcpdumpMonitor:
+    def __init__(self, api_key):
+        self.api_key = api_key
+
```
Comment (storkinsj) [inline honeypod/TcpdumpMonitor.py:36]: Please pull in necessary code from main. Our interface and our IP address can change. Main already detects this:

#
# Get my interface; default eth0
#
network_interface = "eth0"
ip_address = ""


net_interfaces = psutil.net_if_addrs()

found = False
for interface, addresses in net_interfaces.items():
    for address in addresses:                                            
        if interface != "lo" and not address.address.startswith("127."): 
            network_interface = interface                                
            ip_address = address.address
            found = True
            break
    if found:
        break

File: honeypod/TcpdumpMonitor.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+            model="gpt-3.5-turbo",
+            messages=[
+
+                {"role": "user", "content": question}
+            ]
+        )
+
+        answer = response.choices[0].message.content
+        print(f"Assistant: {answer}")
+        command = 'echo "{}" >> {}'.format(answer, path_of_log)
```
Comment (storkinsj) [inline honeypod/TcpdumpMonitor.py]: The format is incorrect.
   command = 'echo "HONEYPOD: Suspicious network traffic: {} {}" etc.
   
   We don't need open AI section, correct?

File: honeypod/TcpdumpMonitor.py [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+        os.system(command) 
+        
+  
+
+# Example usage
+def main():
+    api_key = "<your-key>"
+    monitor = TcpdumpMonitor(api_key)
+    tcpdump_program_path = "/usr/bin/tcpdump"
+    log_path = "/var/log/honeypod.log"
```
Comment (storkinsj) [inline honeypod/TcpdumpMonitor.py]: Please pull necessary code from main branch this needs to be readable from environment.
if "honeylog" in os.environ:
    honeylog = os.getenv(log_env_name)
else:
    honeylog = '/var/log/honeypod'

File: honeypod/TcpdumpMonitor.py [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+    interface = "eth0"
+    tcpfilter = f"not src host 172.17.0.2"
```
Comment (storkinsj) [inline honeypod/TcpdumpMonitor.py]: IP address can not be hard coded.

**Review by storkinsj — COMMENTED** [review]
Many of the improvements to p0f monitor are missing we need to get those into this code.

File: honeypod/TcpdumpMonitor.py (hunk 1 above)
Comment (cklonger) [reply honeypod/TcpdumpMonitor.py:6]: added import

File: honeypod/TcpdumpMonitor.py (hunk 5 above)
Comment (cklonger) [reply honeypod/TcpdumpMonitor.py]: updated

**Pull request closed without merging**